src/common/options.cc

```diff
@@ -1009,6 +1009,10 @@ std::vector<Option> get_global_options() {
     .set_default(100_M)
     .set_description("Limit messages that are read off the network but still being processed"),
 
+    Option("ms_dispatch_throttle_log_interval", Option::TYPE_INT, Option::LEVEL_ADVANCED)
```

Comment:
```suggestion
    Option("ms_dispatch_throttle_log_interval", Option:: TYPE_INT, Option::LEVEL_ADVANCED)
```

Reply:
@joscollin why did you mark this comment resolved?

Reply:
@tchaikov As you can see above, github doesn't show any difference in your Suggested change (hover the mouse on the `Commit suggestion` button says `This suggestion is invalid`). I couldn't see any difference too. I think this happened because you had edited the Suggestion later?

Anyway, please let me know your review comment again. Thanks.

Reply:
@joscollin updated my comment. PTAL.